packages/google_adsense/lib/src/ad_unit_params.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Contains adUnitParams keys constants for convenience and documentation
+class AdUnitParams {
```

Comment:
Hi @ditman! Going over it again, Im not sure camelCase naming will bring any value (because there is not pure JS API for AdSense, rather data-attributes in the html tag). At the same time SCREAMING_SNAKE_CASE suggests that these are constants so Im not changing this for now